### DISCUSSION TITLE
qexo里面发布了文章，但是hexo里面没有文章

### DISCUSSION THREAD

**Discussion by kiocou** [Qexo/Qexo#435] [General]
![批注 2023-08-24 111356](https://github.com/Qexo/Qexo/assets/89248151/66f8ffb1-f4a6-4dcb-b677-ac81c6f09bf4)

qexo里面发布了文章，但是hexo里面没有文章
![批注 2023-08-24 111331](https://github.com/Qexo/Qexo/assets/89248151/f4605a34-9caa-49cd-b671-78c5dd593688)

**Comment by kiocou** [comment]
没有显示”测试“文章

**Comment by kiocou** [comment]
缓存也删除了

**Comment by am-abudu** [comment]
未配置自动部署

**Comment by Shiguang-coding** [comment]
我也遇到了相同的问题，不会是不在一个仓库的原因吧，博客默认用github的pages功能部署，仓库是 xxx.github.io，但是qexo关联的仓库是存放源码的目录，发布之后源码目录确实更新了，但是博客没变化，当然Hexo部署命令 hexo d 默认是只上传public目录下的静态文件的，我不知道qexo是怎么关联的，难道每次发布新文章后需要从源码仓库拉取代码到本地重新执行hexo g 生成静态文件再发布到pages部署的仓库吗，刚了解qexo,不知道怎么用的，文档里也没有具体使用说明

**Reply by am-abudu** [reply]
请查看参考文献部分 完成自动化部署 https://www.oplog.cn/qexo/configs/provider.html#github-%E4%BB%93%E5%BA%93

**Reply by Shiguang-coding** [reply]
配置了自动化部署，构建也成功了，还是没有生效
![image](https://github.com/Qexo/Qexo/assets/75553284/a16dd4d6-db5c-48ff-a504-9a4dbb9239c2)

这是我配置的操作过程，专门写了文章记录，是不是哪里配的不对:https://blog.shiguang666.eu.org/2024/06/24/49078ab077e1
另外我还写了份部署文档： https://blog.shiguang666.eu.org/2024/06/22/0418c76cd199

现在想想还要配置图床，自己搭的话又有点麻烦，可能还要考虑后期迁问题，还是先在本地写吧，修改和管理的话可以用面板工具

**Reply by kiocou** [reply]
好的👌🏻谢谢了
